node/cli/src/export_metadata_cmd.rs

```diff
@@ -54,4 +53,9 @@ impl CliConfiguration for ExportMetadataCmd {
 	fn shared_params(&self) -> &SharedParams {
 		&self.shared_params
 	}
+
+	// We never want to use any stored data. Always just use fresh.
+	fn base_path(&self) -> Result<Option<sc_service::BasePath>, sc_cli::Error> {
+		Ok(Some(sc_service::BasePath::new_temp_dir()?))
```

Comment:
The actual fix!

Reply:
💯

Reply:
🎉

Reply:
Is `base_path` called somewhere?

Reply:
It is. The `impl CliConfiguration` trait is used deep down.